### PR TITLE
Make Game2048.grid private and add a getter

### PR DESCRIPTION
**Pull request by davidkwlam**

### CODE REVIEW

File: src/game/game2048.ts [hunk 1]
```diff
@@ -27,13 +27,17 @@ export interface IGameState {
...
+  private grid: Grid;
   private userActionsQueue: Action[] = [];
 
   constructor(size: number, private rand: IRandom) {
     this.grid = new Grid(size);
   }
 
+  public getGrid() {
+    return this.grid;
+  }
```
Comment (davidkwlam) [inline src/game/game2048.ts:39]: Putting `grid` behind a getter is safer

**Comment by dennispiUB** [comment]
A comment here!

File: src/game/game2048.ts (hunk 1 above)
Comment (dennispiUB) [reply src/game/game2048.ts:39]: I'll start from ehre instead!

--

Comment posted using [Unblocked](https://getunblocked.com/dashboard/team/461c33ed-2d8d-468d-9409-d2cec26e4741/thread/305951ef-808e-4f06-9d8e-64f58b58af82?message=09cfa182-b592-4be0-8758-6941d151926c).

Comment (dennispiUB) [reply src/game/game2048.ts:39]: Ok, try this from the comment section!

**Pull request merged**